### PR TITLE
Update Readme - how use commandline tool

### PR DESCRIPTION
**Pull request by salem84**
Added description of tool and usage (partially taken from telerik site)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ JustAssembly is a lightweight .NET assembly diff and analysis tool built on top
 
 ![alt text](https://d585tldpucybw.cloudfront.net/sfimages/default-source/productsimages/justassembly/how-it-works.png)
 
+## Command line usage
+
+It is possible to use JustAssembly with command line interface, in order to generate an XML file with assembly differences.
```
Comment (adimitrov93) [inline README.md]: To be more accurate - "assembly API differences".

File: README.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ JustAssembly is a lightweight .NET assembly diff and analysis tool built on top
...
+
+That effectively makes it possible to use JustAssembly in an automated build pipeline to analyse differences between the output of the previous and the latest build (Semantic Versioning in future release, see issue #14 for details).
```
Comment (adimitrov93) [inline README.md]: Let's make "#14" a hyperlink. Otherwise it is not that helpful.

File: README.md (hunk 1 above)
Comment (salem84) [reply README.md]: Yes...I totally agree with you

Comment (salem84) [reply README.md]: Absolutely I agree with you

File: README.md (hunk 2 above)
Comment (salem84) [reply README.md]: It is correct with telerik/JustAssembly#14 ? I wrote only #14 but link doesn't work in my repos

Comment (adimitrov93) [reply README.md]: Yes, it should work properly when merged into this repo. It does not work when in your repo, because there is no issue/pr #14

**Pull request merged**